assets/css/_components.scss

```diff
@@ -0,0 +1,48 @@
+/* Buttons */
+
+.btn--likecoin {
```

Comment:
Just to confirm, we are using http://getbem.com/naming/ 

Right?

Reply:
Yes =)